engine/apps/alerts/models/custom_button.py

```diff
@@ -75,6 +75,7 @@ class CustomButton(models.Model):
     password = models.CharField(max_length=100, null=True, default=None)
     deleted_at = models.DateTimeField(blank=True, null=True)
     authorization_header = models.CharField(max_length=1000, null=True, default=None)
+    headers = models.TextField(null=True, default=None)
```

Comment:
did you consider using something like the following?
```python3
headers = models.JSONField(null=True, default=None)
```
this _may_ make the serialization/deserialization and validation simpler (would need to test)